### ISSUE TITLE
Remove `.breadcrumb` overrides that enforce text clipping

### ISSUE THREAD

**Issue by cjg89** [UCF/Main-Site-Theme#353]
**Describe the bug**

https://github.com/UCF/Main-Site-Theme/blob/ba14233e19e3dd253e6584628a25049d3311defd/src/scss/components/_breadcrumbs.scss

The style overrides in this file should be removed so that Athena breadcrumbs are allowed to extend onto more than one line.

**Steps to Reproduce**

n/a

**Expected Behavior**

Breadcrumbs should be able to flow onto multiple lines to avoid text clipping when zoomed to 200%.

**Possible Fix**

Remove the styles linked above and make adjustments as needed where breadcrumbs are in use (should just be the single degree template).

**Your environment:**

n/a

**Additional context**

We get flagged in SiteImprove for breadcrumb content clipping if said content clips in a way that differs from 100%/standard zoom vs 200% zoom.
